recipes/pika/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.10.0" %}
+
+package:
+  name: pika
+  version: {{ version }}
+
+source:
+  fn: pika-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pika/pika-{{ version }}.tar.gz
+  sha256: 7277b4d12a99efa4058782614d84138983f9f89d690bdfcea66290d810806459
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - pika
+
+about:
+  home: http://pika.readthedocs.org
+  license: BSD 3-clause
+  summary: Pure Python RabbitMQ/AMQP 0-9-1 client library
+
+extra:
+  recipe-maintainers:
+    - frol
+    - mgr
+    - tonyg
+    - vitaly-krugl
```

Comment:
@jakirkham They are listed in [Pika Github organization](https://github.com/orgs/pika/people)